tensorflow/python/framework/ops_test.py

```diff
@@ -1308,6 +1308,18 @@ def testBasics(self):
     self.assertEqual("bar_2", g.unique_name("bar", mark_as_used=False))
     self.assertEqual("bar_2", g.unique_name("bar"))
 
+  def testBackslashRegex(self):
```

Comment:
Please also include a string that contains a dash character (adding it to the tests below is fine).

Reply:
Thanks @mdanatg, added `-` into part of the tested string.